cmd/notary/delegations.go

```diff
@@ -171,9 +179,17 @@ func (d *delegationCommander) delegationRemove(cmd *cobra.Command, args []string
 	if d.removeAll {
 		cmd.Printf("Forced removal (including all keys and paths) of delegation role %s to repository \"%s\" staged for next publish.\n", role, gun)
 	} else {
-		cmd.Printf(
-			"Removal of delegation role %s with keys %s and paths %s, to repository \"%s\" staged for next publish.\n",
-			role, keyIDs, d.paths, gun)
+		removingItems := ""
+		if len(keyIDs) > 0 {
+			removingItems = removingItems + fmt.Sprintf("with keys %s, ", keyIDs)
+		}
+		if d.allPaths {
+			removingItems = removingItems + "with all paths,"
```

Comment:
Non-blocking nitpick - should this string terminate with a space like the other ones?  Alternately should none of them have a trailing space and the format string should just include the space?